src/main/perl/Log.pm

```diff
@@ -26,75 +41,70 @@ CAF::Log - Simple class for handling log files
 
   use CAF::Log;
 
-  my $log=CAF::Log->new('/foo/bar','at');
+  my $log = CAF::Log->new('/foo/bar', 'at');
 
   $log->print("this goes to the log file\n");
   $log->close();
 
-=head1 INHERITANCE
-
-  CAF::Reporter
-
 =head1 DESCRIPTION
 
 The B<CAF::Log> class allows to instantiate objects for writing log files.
 A log file line can be prefixed by a time stamp.
 
-
-=over
-
-=cut
-
-#------------------------------------------------------------
-#                      Public Methods/Functions
-#------------------------------------------------------------
-
-=pod
-
-=back
-
 =head2 Public methods
 
 =over 4
 
-=item close(): boolean
+=item C<close()>: boolean
 
-closes the log file.
+closes the log file, returns SUCCESS on succes, undef otherwise
+(if no FH attribute exists).
 
 =cut
 
-sub close ($) {
-  my $self=shift;
+# Called during DESTROY, use $_XYZ flavour
+sub close ($)
```

Comment:
i'd prefer to do that outside of thei work, i opened https://github.com/quattor/CAF/issues/114